api-docs.templ

```diff
@@ -0,0 +1,104 @@
+# API Documentation
```

Comment:
This file comes from https://github.com/pseudomuto/protoc-gen-doc/blob/master/resources/markdown.tmpl. MIT license.

The title differs from the original. Placing our own copy in this repo gives us more control over how information is laid out in the final rendered file.